simulation/src/main/java/com/palantir/dialogue/core/Benchmark.java

```diff
@@ -262,6 +262,7 @@ public void onFailure(Throwable throwable) {
                                     .response()
                                     .channelName(SimulationUtils.CHANNEL_NAME)
                                     .serviceName(SimulationUtils.SERVICE_NAME)
+                                    .endpoint(DEFAULT_ENDPOINT.endpointName())
```

Comment:
Ok, it looks like there are tests that have multiple endpoints. So need to figure out how to get the total here.

Reply:
Options I can see:

1. Looks like we only look at mean, I can calculate that.
2. Make the test files be endpoint aware?

Reply:
I defer to your judgement -- In some places we've added benchmark-specific metrics, so that's another option if it makes things easier.

Reply:
Ok what I have works, so I think that's fine for now. We have unit tests in TimingChannel that actually test the metrics; I'd maybe consider just returning the mean on the BenchmarkResult, instead of a whole metric snapshot, but not too fussed.